src/xcode/ENA/ENA/Source/Views/AccessibleTabbarController.swift

```diff
@@ -0,0 +1,46 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+final class AccessibleTabbarController: UITabBarController {
```

Comment:
I think we should discuss this further. Maybe in the DevCamp. I think we should follow the platform conventions here not scale the content of bars.

Reply:
I think it is the platform convention since it's officially documented by Apple: 

> In iOS v5.0 and later, you can customize the appearance of tab bars by setting item label text attributes using appearance selectors declared by UIBarItem. You can also use the methods listed in Customizing the Item's Appearance. You can customize the appearance of all tab bar items using the appearance proxy (for example, [UITabBarItem appearance]), or just of a single tab bar item.
By default, unselected and selected images are automatically created from the alpha values in the source images. To prevent system coloring, provide images with UIImage.RenderingMode.alwaysOriginal.

Reply:
You are talking about API availability, I'm talking about platform conventions. Show me one Apple app with a scaling tab bar 😉

Reply:
As I noted below the tab bar itself does not scale. Only the title of the tabs.

Reply:
I understand that. But the platform convention is not to scale it, because there already is an existing accessibility feature built in that covers that. It was even made available for custom UI components in form of the large content viewer starting from iOS 13, It's definitely the recommended way by Apple to do this: https://developer.apple.com/videos/play/wwdc2019/261/

Reply:
And that is still available.

Reply:
BTW the API are linked to was introduced before dynamic type, so it is technically possible to use it with scaling fonts, but was not intended when it was introduced.

Reply:
I have put it on the agenda for Tuesday :)